bundle/Controller/UserRegisterController.php

```diff
@@ -7,74 +7,11 @@
  */
 namespace EzSystems\RepositoryFormsBundle\Controller;
 
-use eZ\Bundle\EzPublishCoreBundle\Controller;
-use eZ\Publish\Core\MVC\Symfony\Security\Authorization\Attribute;
-use EzSystems\RepositoryForms\Data\Mapper\UserRegisterMapper;
-use EzSystems\RepositoryForms\Form\ActionDispatcher\ActionDispatcherInterface;
-use EzSystems\RepositoryForms\Form\Type\User\UserRegisterType;
-use EzSystems\RepositoryForms\User\View\UserRegisterConfirmView;
-use EzSystems\RepositoryForms\User\View\UserRegisterFormView;
-use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\HttpKernel\Exception\UnauthorizedHttpException;
+use EzSystems\EzPlatformUserBundle\Controller\UserRegisterController as BaseUserRegisterController;
 
-class UserRegisterController extends Controller
+/**
+ * @deprecated Deprecated in 1.5 and will be removed in 2.0. Please use \EzSystems\EzPlatformUserBundle\Controller\UserRegisterController instead.
```

Comment:
I think the version is wrong here. It should be 2.5 and 3.0

Reply:
Yup, it is 1.5 and 2.0 for adminUI, my bad.